### PR TITLE
PDE-3152 feat(cli) - Update `team:get` CLI command to fetch Collaborator.

### PR DESCRIPTION
**Pull request by standielpls**
JIRA: https://zapierorg.atlassian.net/browse/PDE-3152

Removes `subscribers` from `team:get` command
Fetches `limited collaborators` for `team:get` command.

### CODE REVIEW

**Comment by xavdid** [comment]
cool, so the code here looks good. The only note is that this as written should be marked as a breaking change. If I have CLI code that reads subscribers, then it would break after this update. eg:

```
zapier team:get --format json | jq '.subscribers'
```

So, either we:

- mark this as a major version and release it when we do that next (probably ~ may, when the new node version rotates)
- use the new endpoint, but output the data under the old key (assuming the structure of what comes out of the API call is otherwise identical)

I'm good either way, just wanted to flag!

**Comment by eliangcs** [comment]
@xavdid that's a good callout! Sorry, I didn't think of people might have workflows relying on the `subscribers` field when writing the ticket [PDE-3152](https://zapierorg.atlassian.net/browse/PDE-3152). I'd suggest we keep it backward compatible. So how about we do these:

- the `team:get` still gets `subscribers` from the `/subscribers` endpoint
- when the time comes, we deprecate the `/subscribers` endpoint by making it return an empty result ([PDE-3161](https://zapierorg.atlassian.net/browse/PDE-3161))

> use the new endpoint, but output the data under the old key (assuming the structure of what comes out of the API call is otherwise identical)

FWIW, `subscriber` and `limited_collaborator` are not the same, so we probably shouldn't use the old key `subscribers` for `limited_collaborators`.

@standielpls does that make sense?

**Comment by standielpls** [comment]
Yep that makes sense! 
As `subscriber` and `limited_collaborator` are not the same, when we display the table, the user can be 1 of 3 roles `admin`, `subscribers` and `collaborator`.

File: packages/cli/src/oclif/commands/team/get.js [hunk 1]
```diff
@@ -48,10 +60,11 @@ class TeamListCommand extends ZapierBaseCommand {
 TeamListCommand.flags = buildFlags({ opts: { format: true } });
 TeamListCommand.description = `Get team members involved with your integration.
 
-These users come in two levels:
+These users come in three levels:
 
   * \`admin\`, who can edit everything about the integration
-  * \`subscriber\`, who can't directly access the app, but will receive periodic email updates. These updates include quarterly health socores and more.
+  * \`subscriber\`, who can't directly access the app, but will receive periodic email updates. These updates include quarterly health scores and more.
+  * \`collaborator\`, who has read-only access for the app, but will receive periodic email updates. These updates include quarterly health scores and more.
```
Comment (eliangcs) [inline packages/cli/src/oclif/commands/team/get.js:67]: Same comment as [this](https://github.com/zapier/zapier-platform/pull/539#discussion_r857359686). Maybe  `collaborator` should go before `subscriber`?

**Review by eliangcs — APPROVED** [review]

**Review by eliangcs — APPROVED** [review]
💯

**Pull request merged**